src/main/java/lukemccon/airdrop/packages/Package.java

```diff
@@ -35,18 +42,18 @@ public double getPrice() {
 		return this.price;
 	}
 
+	public String getName() { return this.name; }
+
 	public Boolean canAfford(Player player) {
-		User user = new User(player, Airdrop.ESSENTIALS);
-		BigDecimal price = new BigDecimal(this.getPrice());
+		double balance = econ.getBalance(player);
+		int comparison = Double.compare(balance, this.price);
 
-		System.out.println(user.getMoney());
-		return user.canAfford(price);
-	}
+        return comparison >= 0;
+    }
 
 	public void chargeUser(Player player) {
-		User user = new User(player, Airdrop.ESSENTIALS);
-		BigDecimal price = new BigDecimal(this.getPrice());
-		user.takeMoney(price);
+		econ.withdrawPlayer(player, this.price);
+		ChatHandler.sendMessage(player, ChatColor.AQUA + "$" + this.price + ChatColor.BLUE + " has been taken from your account");
```

Comment:
Do we want to use Aqua and Blue in the same message?

Reply:
I guess we can abstract it out to a primary and secondary color scheme in the future